.github/workflows/rust.yml

```diff
@@ -0,0 +1,22 @@
+name: coverage
+
+on: [push]
+jobs:
+  test:
+    name: coverage
+    runs-on: ubuntu-latest
+    container:
+      image: xd009642/tarpaulin:develop-nightly
```

Comment:
for convenience I've used the Github workflow from `tarpaulin` crate, but post migration to `github.com/radixdlt/Sargon` we should probably use same as RET, and can investigate if we should use [`grcov`](https://github.com/mozilla/grcov) instead of Tarpaulin